### PR TITLE
feat: [PL-27237]: Globally enable CUSTOM_RESOURCEGROUP_SCOPE and INHE…

### PR DESCRIPTION
**Pull request by reetika04**
…RITED_USER_GROUP

### Summary

Removing the feature flags CUSTOM_RESOURCEGROUP_SCOPE and INHERITED_USER_GROUP

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10391)

File: src/modules/10-common/featureFlags.ts [hunk 1]
```diff
@@ -65,9 +65,8 @@ export enum FeatureFlag {
   ENV_GROUP = 'ENV_GROUP',
   NEW_PIPELINE_STUDIO = 'NEW_PIPELINE_STUDIO',
   NG_SVC_ENV_REDESIGN = 'NG_SVC_ENV_REDESIGN',
-  CUSTOM_RESOURCEGROUP_SCOPE = 'CUSTOM_RESOURCEGROUP_SCOPE',
+  CLOUDFORMATION = 'CLOUDFORMATION_NG',
```
Comment (upadhyay-an) [inline src/modules/10-common/featureFlags.ts:68]: Intentional change?

**Review by upadhyay-an — APPROVED** [review]
Please add description/summary to PR that it's about removing to feature flags.

**Comment by reetika04** [comment]
retrigger coverage

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by upadhyay-an — APPROVED** [review]

**Pull request merged**